### PR TITLE
Add formatting scheme to wiki

### PR DESCRIPTION
**Pull request by PoignardAzur**

### CODE REVIEW

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]
Shaders are an exception to the licensing here and are important enough to Vello that it is probably worth calling that out specifically.

File: content/wiki/formatting_scheme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+The year should be the file's creation year.
+"XXXX" should be the project's name.
+The license depends on the project and the file type, but is usually "either Apache 2.0 or MIT License, whichever is least restrictive".
+
+## Rust
+
+Most Linebender projects files are written in Rust.
+For those files, the default formatting produced by `rustfmt` is fine.
```
Comment (xStrom) [inline content/wiki/formatting_scheme.md:21]:
```suggestion
For those files, the formatting produced by `rustfmt` is required.
```

1. It's usually default, but may not be if we introduce `rustfmt.toml` like we have for Druid.
2. The use of *fine* implies there's a different option. However it's actually a strict requirement that the formatting matches what `rustfmt` produces.

File: content/wiki/formatting_scheme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Markdown
+
+In Markdown files, every paragraph should have one line per sentence. This makes git diff easier to read.
```
Comment (xStrom) [inline content/wiki/formatting_scheme.md:25]: Might be worth linking to:
* https://nick.groenen.me/notes/one-sentence-per-line/
* https://sive.rs/1s

As additional resources here for the rationale.

Comment (DJMcNab) [inline content/wiki/formatting_scheme.md:25]: Is this an intentional joke, or just an amusing accident (that these two sentences are in one line?)

I would accept this being the only exception to that rule, but I don't think it's possible to document that, so I think we should fix it

Comment (PoignardAzur) [reply content/wiki/formatting_scheme.md:25]: Accident. Seriously, this rule is so annoying to apply. I don't know any auto-formatter that applies it.

**Comment by PoignardAzur** [comment]
I added small changes to address feedback.

**Review by xStrom — APPROVED** [review]
Thanks! 🚀

**Review by waywardmonkeys — APPROVED** [review]

**Pull request merged**